ports/lcms/patches/0001-Add-CMake-build.patch

```diff
@@ -0,0 +1,100 @@
+From 5364f7e68cc911bba8bc6fdaec8326caa803afdb Mon Sep 17 00:00:00 2001
+From: Don <don.j.olmstead@gmail.com>
+Date: Thu, 11 Nov 2021 14:31:22 -0800
+Subject: [PATCH] Add CMake build
+
+---
+ CMakeLists.txt | 81 ++++++++++++++++++++++++++++++++++++++++++++++++++
+ 1 file changed, 81 insertions(+)
+ create mode 100644 CMakeLists.txt
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..5325d0a
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,81 @@
++cmake_minimum_required(VERSION 3.15)
++
++project(little-cms
++    VERSION 2.12.0
++    LANGUAGES C
++)
++
++include(CheckSymbolExists)
++include(GNUInstallDirs)
++
++add_library(lcms2 "")
++target_sources(lcms2 PRIVATE
++    src/cmsalpha.c
```

Comment:
It'd be nice to get this list sorted.

Reply:
I was just using the listing in the Meson port but I'll go ahead and sort it all.